utils/go-client/metric.go

```diff
@@ -0,0 +1,217 @@
+package go_client
```

Comment:
go-client is pretty generic name for the package. What's being offered is a transport wrapper helper around kubernetes rest client. Maybe `utils/kubeclientmetrics` ?